packages/flet/lib/src/controls/circle_avatar.dart

```diff
@@ -40,6 +43,12 @@ class CircleAvatarControl extends StatelessWidget {
         radius: control.attrDouble("radius"),
         minRadius: control.attrDouble("minRadius"),
         maxRadius: control.attrDouble("maxRadius"),
+        onBackgroundImageError: (object, trace) {
+          backend.triggerControlEvent(control.id, "imageError", "background");
+        },
+        onForegroundImageError: (object, trace) {
+          backend.triggerControlEvent(control.id, "imageError", "foreground");
+        },
```

Comment:
For circle avatar it would be also nice to support images from assets, because right now it's just `NetworkImage()`.

Reply:
Added.